exporters/inmemory/src/main/java/io/opentelemetry/exporters/inmemory/InMemoryTracing.java

```diff
@@ -43,71 +45,59 @@
  *
  * @since 0.1.0
  */
-public final class InMemoryTracing {
-  private final TracerSdkProvider tracerSdkProvider;
-  private final InMemorySpanExporter exporter;
-
+@AutoValue
+@Immutable
+public abstract class InMemoryTracing {
```

Comment:
@jkwatson I need your brain to help me name this class :) Idea is that probably I will have an equivalent for Metrics as well, also every exporter should offer this helper to configure the span exporter class.

Reply:
Personally I think the name is fine, but if you manage to find a better name, I'm in ;)

Reply:
I think this name works fine. :shipit: